app/controllers/concerns/fields.rb

```diff
@@ -1,8 +1,8 @@
 module Fields
   private
 
-  def field_params
-    return {} unless params[:fields]
+  def field_params(default = {})
+    return default unless params[:fields]
```

Comment:
long term we probably should preserve default keys that aren't specified as a `fields[{key}]` parameter in the request.

But for now just being able to override the default fields is enough for the current use-case.